pallets/pallet-asset-switch/src/mock.rs

```diff
@@ -186,37 +180,9 @@ impl crate::Config for MockRuntime {
 	type XcmRouter = AlwaysSuccessfulXcmRouter;
 }
 
-#[derive(Clone)]
-pub(crate) struct NewSwitchPairInfo {
-	pub(crate) circulating_supply: u128,
-	pub(crate) pool_account: AccountId32,
-	pub(crate) remote_asset_id: VersionedAssetId,
-	pub(crate) remote_fee: VersionedMultiAsset,
-	pub(crate) remote_reserve_location: VersionedMultiLocation,
-	pub(crate) status: SwitchPairStatus,
-	pub(crate) total_issuance: u128,
-	pub(crate) min_remote_balance: u128,
-}
-
-impl From<NewSwitchPairInfo> for SwitchPairInfoOf<MockRuntime> {
```

Comment:
Ah I see, u replaced this implementation with `from_input_unchecked`? May I ask why?

Reply:
I answered in [this comment above](https://github.com/KILTprotocol/kilt-node/pull/674#discussion_r1687988782). It's mostly a matter of semantics, nothing really technical.